.github/workflows/docs.yml

```diff
@@ -0,0 +1,65 @@
+name: Deploy docs
+
+on:
+  push:
+    branches: ["master"]
+    paths:
+      - "docs/**"
+  pull_request:
+    branches: ["master"]
+    paths:
+      - "docs/**"
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: docs
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.1' # Not needed with a .ruby-version file
```

Comment:
There is .ruby-version file. Is there a reason to keep this variable?

Reply:
lol good catch - even funnier that such comment is coming from the readme of setup-ruby action